api/v1alpha1/wavefrontalert_types.go

```diff
@@ -88,11 +94,14 @@ const (
 type State string
 
 const (
-	Ready    State = "Ready"
-	Error    State = "Error"
-	Creating State = "Creating"
-	Updating State = "Updating"
-	Deleting State = "Deleting"
+	Ready               State = "Ready"
```

Comment:
what's the difference between Ready and ReadyToBeUsed

Reply:
ReadyTobeUsed- if the wavefront alert is templatized and can be ready to be used via AlertsConfig
Ready- Stand alone wavefront alert spec which created an alert in wavefront successfully